OsmAnd/src/net/osmand/plus/views/POIMapLayer.java

```diff
@@ -339,7 +339,7 @@ public PointDescription getObjectName(Object o) {
 				if (Algorithms.isEmpty(preferredLang)) {
 					preferredLang = app.getLanguage();
 				}
-				preferredLang = WikipediaPoiMenu.getWikiArticleLanguage(app,
+				preferredLang = OsmandPlugin.onGetMapObjectsLocale(
```

Comment:
should be method 
OsmandPlugin.onGetMapObjectsLocale(Amienity, preferredLang)

So each plugin can deal with different object types